File: Year 3/MA/BookRentals/Models/Book.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace BookABook.Models
{
    public class Book
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string Author { get; set; }

        public string PublishDate { get; set; }

        public bool IsBooked { get; set; }

        public Guid UserId { get; set; }
        
        public byte[] Image { get; set; }
    }
}

File: Year 3/MA/BookABook/app/src/main/java/com/example/bookabook/todo/data/Book.kt
package com.example.bookabook.todo.data

data class Book(
    val id: Int,
    var name: String,
    var author: String,
    var publishDate: String,
    var isBooked: String,
    var image: String,
) {
    override fun toString(): String = "$id, $name, $author, $publishDate" + if (isBooked == "true") " yes" else " no"
}
